extra_data/export.py

```diff
@@ -216,7 +218,17 @@ def feed(self, data, metadata=None):
         self._buffer.put(self._serialize(data, metadata))
 
 
-def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
+def gen_time():
+    """Create a mock timestamp for meta-data"""
+    timestamp = time()
+    sec, frac = str(timestamp).split('.')
+    frac = frac.ljust(18, '0') # attosecond resolution
+    return sec, frac
```

Comment:
We should either remove this function again, or make use of it in ZMQStreamer.

Reply:
Removed